### PR TITLE
Consistent Engauge Description

### PR DESCRIPTION
**Pull request by TobiasWinchen**
Hi,

there are currently several different short-descriptions of engauge for the
different Linux distributions. I tried to merge the description in the debian
packages and the appdata information and the information on the homepage, to
give a consistent representation, and added the .desktop file from the debian
packages, so it can be reused for other packages. 

Cheers,

Tobi

### CODE REVIEW

File: dev/engauge-digitizer.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Engauge-digitizer
```
Comment (crayxt) [inline dev/engauge-digitizer.desktop]: probably Engauge Digitizer, per https://markummitchell.github.io/engauge-digitizer ?

File: dev/gnome/engauge-digitizer.appdata.xml [hunk 2]
```diff
@@ -5,41 +5,42 @@
...
-      </li>
-      <li>
-        You are a webmaster with visitor statistics charts and you
-        want to do statistical analyses
-      </li>
-    </ul>
+		<p>
+			The Engauge Digitizer tool assists in interactively extracting numbers
+			from images of graphs. Conceptually, it is thus the opposite of a
+			graphing tool that converts data points to graphs. It provides asistance
```
Comment (crayxt) [inline dev/gnome/engauge-digitizer.appdata.xml]: asistance -> assistance

File: dev/gnome/engauge-digitizer.appdata.xml [hunk 3]
```diff
@@ -5,41 +5,42 @@
...
+			in enhancing the image quality and matching the data points. Engauge
+			Digitizer is used by individuals such as grad students and researchers as
+			well engineers and employeers in large government and commercial
```
Comment (crayxt) [inline dev/gnome/engauge-digitizer.appdata.xml]: employeers -> employers ?

File: dev/gnome/engauge-digitizer.appdata.xml [hunk 4]
```diff
@@ -5,41 +5,42 @@
...
+			organizations for processing single files  but also managing databases of
```
Comment (crayxt) [inline dev/gnome/engauge-digitizer.appdata.xml]: Double white space between `files` and `but`

File: dev/gnome/engauge-digitizer.appdata.xml [hunk 5]
```diff
@@ -5,41 +5,42 @@
...
+			<li> Automatic grid line removal</li>
+			<li> Automatic point and axis matching</li>
+			<li> Automatic curve tracing</li>
+			<li> Image processing for separating important details from background information </li>
+			<li> A wizard providing an interactive tutorial to explain the basic steps </li>
+			<li> Multiple coordinate systems in the same image can be digitized in advanced mode</li>
+			<li> Cubic spline interpolation between points gives more accurate curves with fewer points </li>
+			<li> Handling of cartesian, polar, linear and logarithmic graphs</li>
+			<li> Support for drag-and-drop and copy-and-paste</li>
+			<li> Context sensitive help, user manual and tutorials explaining every feature</li>
```
Comment (crayxt) [inline dev/gnome/engauge-digitizer.appdata.xml]: Context sensitive -> Context-sensitive

File: dev/gnome/engauge-digitizer.appdata.xml [hunk 6]
```diff
@@ -5,41 +5,42 @@
...
+			<li> Preview windows to give immediate feedback while modifying settings</li>
+			<li> Automated line and point extraction to rapidly digitizes data </li>
```
Comment (crayxt) [inline dev/gnome/engauge-digitizer.appdata.xml]: rapidly digitizes -> rapidly digitizing?

File: dev/gnome/engauge-digitizer.appdata.xml [hunk 7]
```diff
@@ -5,41 +5,42 @@
...
+			graphing tool that converts data points to graphs. It provides asistance
+			in enhancing the image quality and matching the data points. Engauge
+			Digitizer is used by individuals such as grad students and researchers as
+			well engineers and employeers in large government and commercial
+			organizations for processing single files  but also managing databases of
+			thousands of image files. 
+		</p>
+			
+		<p>
+			Features of engauge include:
```
Comment (crayxt) [inline dev/gnome/engauge-digitizer.appdata.xml]: engauge -> Engauge Digitizer

File: dev/gnome/engauge-digitizer.appdata.xml [hunk 8]
```diff
@@ -5,41 +5,42 @@
...
+		</p>
+		<ul>
+			<li> Automatic grid line removal</li>
```
Comment (crayxt) [inline dev/gnome/engauge-digitizer.appdata.xml]: I would rather delete whitespaces after `<li>` and before `</li>` elements in whole `<ul>` block

**Comment by markummitchell** [comment]
Thank you!

**Comment by crayxt** [comment]
But what about my comments here?

**Comment by markummitchell** [comment]
Yes, they are good comments - I just missed them. I will personally add them this evening. Thanks!

**Comment by TobiasWinchen** [comment]
I included the cmments. Thank you!

**Pull request merged**